### DISCUSSION TITLE
Announcing the v2.36.18 Release

### DISCUSSION THREAD

**Discussion by tinaboyce** [Versent/saml2aws#1356] [Announcements]
Hey!!!

I am so very happy to announce the release of saml2aws [v2.36.18](https://github.com/Versent/saml2aws/releases/tag/v2.36.18)!!! Lots of hard work has been pouring into this project, it is very much appreciated by all who is using this tool. You can see the summary of the changes made here: [v2.36.17...v2.36.18](https://github.com/Versent/saml2aws/compare/v2.36.17...v2.36.18).

Here's a list of the great work from the community that made it into this release:

* Fix pipeline failed to deploy windows pkg due to missing flag
* Move chocolatey deployment to GitHub actions
* Implement assume-time policy limiting
* Support the new flow used in Authentik
* Pass mfa-token in AzureAD provider

A massive thanks to the first-time contributors whose PR made it into this release:
* @eldondevat
* @mokrinsky
* @hejfelix

A massive thanks to @davidobrien1985 for spending some time on the handover of the deployment of windows packages.

Finally, a huge, massive shout-out to @mapkon for your time and continuous effort in maintaining this project.

**Comment by hejfelix** [comment]
When I click the release it doesn't seem like my MFA change is included?

**Reply by hejfelix** [reply]
https://github.com/Versent/saml2aws/issues/902#issuecomment-2487038950
@tinaboyce

**Reply by tinaboyce** [reply]
@hejfelix I have looked into it and it was a bad release with lots of missing PR. Apologies for that as it is an error on my end. Can confirm this PR is part of the current Release https://github.com/Versent/saml2aws/releases/tag/v2.36.19 
![Screenshot 2025-03-14 at 9 21 24 am](https://github.com/user-attachments/assets/18ff0021-9cb5-4728-976c-9e1e84abe379)

**Reply by hejfelix** [reply]
Awesome. Thanks for a great project!
